larreco/WireCell/CellTree_module.cc

```diff
@@ -228,7 +231,9 @@ namespace wc {
     fSaveSimChannel = p.get<bool>("saveSimChannel");
     fSaveTrigger = p.get<bool>("saveTrigger");
     fSaveJSON = p.get<bool>("saveJSON");
+    fSaveJSON_T = p.get<bool>("saveJSON_T");
```

Comment:
Would it ever make sense for a user to specify `saveJSON: false` and `saveJSON_T: true`?  If not, then there should be a check that such a configuration is never used—i.e. if a user specifies that combination, then an exception is thrown.

Reply:
Thanks. I changed the code and configuration.